src/components/reviews-by-category.jsx

```diff
@@ -1,4 +1,41 @@
-const ReviewsByCategory = () => {
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getReviews } from "../utils/api";
+import createReviewCard from "../utils/create-review-card";
 
-}
-export default ReviewsByCategory
+const ReviewsByCategory = (props) => {
+  const [currReviews, setCurrReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { category } = useParams();
+  const { currCategories } = props;
+
+  const categoryObj = currCategories.filter((elem) => {
+    return elem.slug === category;
+  })[0];
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getReviews().then((data) => {
+      const selectedReviews = data.reviews.filter((review) => {
+        return review.category === category;
+      });
+      setCurrReviews(selectedReviews);
```

Comment:
you should not be using a javascript filter in your front end app!

This is too heavy for clients devices (think slow mobile phones, laptops) to handle

Does you API have a query you could use to filter the data?